### PR TITLE
Feat/oibus loki log proxy

### LINKED ISSUE #1325: [Loki logs] - When behind firewalls, make it so that loki logs can be sent to another OIBus
And manage some local forwarding data to display on the oibus

### PR DESCRIPTION
**Pull request by burgerni10**
ON TOP OF #1639 

Resolve #1325

### CODE REVIEW

**Review by jfhenon — COMMENTED** [review]
I will review this one when the silly to trace change will be complete. let me know

File: src/index.js [hunk 1]
```diff
@@ -19,22 +19,23 @@ const MAX_INTERVAL_MILLISECOND = 30 * 1000
...
+
 logger.changeParameters({
-  engineName: 'OIBus-main-thread',
+  engineName: 'OIBus-main',
   logParameters: {
     consoleLog: { level: 'debug' },
     fileLog: {
       level: 'debug',
-      fileName: './main-thread-journal.log',
+      fileName: `${path.parse(configFile).dir}/main-thread-journal.log`,
```
Comment (jfhenon) [inline src/index.js:33]: should the name be OIBus-main-journal.log?

File: src/server/controllers/logController.js [hunk 2]
```diff
@@ -22,4 +22,52 @@ const getLogs = async (ctx) => {
...
+          case 'warn':
+            ctx.app.logger.warn(formattedLog)
+            break
+
+          case 'error':
+            ctx.app.logger.error(formattedLog)
+            break
+
+          default:
+            break
```
Comment (jfhenon) [inline src/server/controllers/logController.js:66]: maybe a warning here?

**Review by jfhenon — COMMENTED** [review]
2 comments

File: src/server/controllers/logController.js (hunk 2 above)
Comment (burgerni10) [reply src/server/controllers/logController.js:66]: So, I rather use console.warn to avoid spamming the logger, what do you think ?

**Pull request merged**